nixos/modules/services/databases/postgresql.nix

```diff
@@ -113,6 +113,17 @@ in
         example = "postgresql_15";
       };
 
+      finalPackage = mkOption {
+        type = types.package;
+        readOnly = true;
+        default = postgresql;
+        defaultText = "with config.services.postgresql; package.withPackages extensions";
+        description = ''
+          The postgresql package that will effectively be used in the system.
+          It consists of the base package with plugins applied to it.
+        '';
+      };
```

Comment:
Assuming you're talking about `internal = true;`: I think it's valuable to have this option in the manual since it's explicitly supposed to be used by people (I stumbled upon this while fixing the postgresql upgrade code at work e.g.) and thus should be documented.